server.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
 
+const path = require('path');
 const app = require('express')();
 const tasksContainer = require('./tasks.json');
+const webpack = require('webpack');
```

Comment:
is it possible to use here `webpack-dev-server`?  in this case, the server will be more focused on API